adhocteam/Head-Start-TTADP
# Add design to Login page

**Description of change**

Make the login page look like the design:

- Implement welcome placard, per design
- Update tests that rely on login placard
- Give that login placard the role=dialog.

**How to test**

Try both logging out, and changing `BYPASS_AUTH=false`. Login page should look like [the design](https://preview.uxpin.com/ad995fd722bd26c0f9a1c8a115f1b41833637308#/pages/134937340/specification/sitemap).

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/67

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
